parcels/field.py

```diff
@@ -320,8 +320,8 @@ def from_netcdf(cls, name, dimensions, filenames, indices={},
                 filebuffer.indslon = indices['lon'] if 'lon' in indices else range(lonsize)
                 filebuffer.indsdepth = indices['depth'] if 'depth' in indices else range(depthsize)
                 for inds in [filebuffer.indslat, filebuffer.indslon, filebuffer.indsdepth]:
-                    if not isinstance(inds, list):
-                        raise RuntimeError('Indices sur field subsetting need to be a list')
+                    if type(inds) not in [list, range]:
```

Comment:
isinstance does not work for more than an argument? I thought we used this already in the code!

Reply:
(just a comment by the way. nothing to do with that)